index.html

```diff
@@ -1,5 +1,21 @@
-<h1>Check it out</h1>
-<a href='hello.html'>hello.html</a><br />
-<a href='hello.php'>hello.php</a><br />
-<a href='hello_db.php'>hello_db.php</a><br />
-
+<html>
+	<head>
+		<title>myBlog</title>
+		<link rel='stylesheet' href='style.css'>
+	</head>
+	<body>
+		<h1>Welcome to my blog!</h1>
+		<div>
+			<h2>Posts</h2>
+			<div>
+				<a href='post_database.php?postID=1'>Post 1</a>
```

Comment:
Eventually you'll want these links to be generated dynamically. Pull all the posts from the db in whatever order you want, loop through them, and print a link for each one. With what you've got here, you still need to update this code anytime a new post is written.